Neos.Eel/Classes/Helper/FileHelper.php

```diff
@@ -61,6 +61,17 @@ public function stat(string $filepath)
         return stat($filepath);
     }
 
+    /**
+     * File exists
```

Comment:
Congrats on your first contribution! I would suggest adding here a better explanation because this will go into the reference documentation

Reply:
```suggestion
     * Check if the given file path exists
```